package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sudomesh-monitor",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
By semver standards this is would reasonably be a minor version bump, not patch https://semver.org/

So hooray, `0.1.0`

Reply:
cool